### ISSUE TITLE
The music from CD track can be played only once，Cannot cycle normally

### ISSUE THREAD

**Issue by tcknight** [joncampbell123/dosbox-x#3010]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar question(s) already exists?

- [X] I have searched and didn't find any similar issues.

### Question

the BGM of the game "Tun Town" could be played through the CD track, but with dosbox-x the bgm cannot cycle normally . I tried the original dosbox 0.74 which the music from CD track could cycle normally, so i wonder if this is some bug or if the problem could be solved by modifying the software settings.

i really like this verson,  so i really hope this problem could be solved , thanks a lot to the great developper~~~

**Comment by Switch123456789** [comment]
I have the same issue with all kinds of CD-images (e.g. GTA1, Network Q RAC Rally & several Audio only CDs), they are stored in cue/bin-format. Most of them were created with [redumper](https://github.com/superg/redumper).

To be precise, the issue is that after a song finished, the playback of the next one seems to be emulated (position and such is reported correctly) but no audio is playing. If I press pause and then play again (in game or in the CD-player software), the audio plays back as expected. So it is ONLY the transition between songs, that breaks things.

It is easy to reproduce and makes many games weird to play (sudden silence), so I am wondering why no one ever noticed or fixed that. If no one can reproduce it, let me know if you need more information.

I found a similar (closed) report, also affecting many games including GTA1: https://github.com/joncampbell123/dosbox-x/issues/2162

**Comment by maron2000** [comment]
GTA1 switched from track 2 to track 3 flawlessly, if this is what you mention. (I didn't try any further)
It's in a bin+cue format with a .bin file for each track.
You may want to show a concrete and a reproducible procedure.

https://github.com/user-attachments/assets/9f105e22-b42c-4425-88ad-5f03f7b9bc08

```
LOG: CDROM: Playing track # 2 0.0 min.-mark
LOG: Started capturing video to: D:\dosbox-x\Release\capture\gta24_000.avi
LOG: CDROM: Playing track # 3 0.0 min.-mark
LOG: Stopped capturing video.
```

**Comment by Switch123456789** [comment]
Sure, I created [this Audio CD image](https://github.com/user-attachments/files/27424563/Audio_CD.zip) for testing. This is what it does in my DOSBox-x:

https://github.com/user-attachments/assets/945e5612-2b8a-4027-9ff7-1bb4ea260af6

This is the only output:
`LOG: CDROM: Playing track # 1 0.0 min.-mark`

**Comment by maron2000** [comment]
~~So you're on Linux?
I don't want to ask many questions until we get on the same track.
You can just open a new issue and fill out the information.~~

**Comment by maron2000** [comment]
Oh yes, playing on a CD player software on DOS didn't go to the next track as well!
Please forget about my previous comment.

https://github.com/user-attachments/assets/e367ee20-2396-4035-b7d8-8cd781a69c26
